sso-kit-hilla-starter/src/main/java/dev/hilla/sso/endpoint/AuthEndpoint.java

```diff
@@ -24,6 +24,8 @@
 @Endpoint
 @AnonymousAllowed
 public class AuthEndpoint {
+    private static final String ROLE_PREFIX = "ROLE_";
+    private static final int ROLE_PREFIX_LENGTH = ROLE_PREFIX.length();
```

Comment:
Is this constant needed if the length is used only once?

Reply:
Not strictly needed, but it's a computed value in a loop. While Java can probably optimize it, I find that more readable.

Reply:
That make sense.